### DISCUSSION TITLE
tutorial/use/ref

### DISCUSSION THREAD

**Discussion by giscus** [yangbo5207/react19feature#21] [General]
# tutorial/use/ref

http://localhost:5173/tutorial/use/ref

**Comment by yangbo5207** [comment]
欢迎各位道友添加我的微信好友 icanmeetu 交流学习心得

**Comment by dlgchg** [comment]
我来了

**Comment by long36708** [comment]
其实早在19年的时候，就有人推荐直接使用props传递ref了，https://github.com/typescript-cheatsheets/react/issues/106
